src/app/file/bmp_format.cpp

```diff
@@ -99,17 +108,25 @@ typedef struct BITMAPFILEHEADER
   uint32_t bfOffBits;
 } BITMAPFILEHEADER;
 
-// Used for both OS/2 and Windows BMP.
+// Used for all Info Header Sizes.
 // Contains only the parameters needed to load the image.
 typedef struct BITMAPINFOHEADER
 {
   uint32_t biWidth;
   uint32_t biHeight;
   uint16_t biBitCount;
   uint32_t biCompression;
+  uint32_t biClrUsed;
+  uint32_t rMask;
+  uint32_t gMask;
+  uint32_t bMask;
+  uint32_t aMask;
+
+  bool isRGBMasks;
+  bool isAlphaMask;
```

Comment:
I think that if the `biSize` is included in the header structure, we could replace these fields as member functions
```suggestion
  bool isRGBMasks() const { ... }
  bool isAlphaMask() const { ... }
```
Not sure if it really worth it.